cluster/saltbase/salt/generate-cert/make-ca-cert.sh

```diff
@@ -18,6 +18,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+if [ "$DEBUG" == "true" ]; then
```

Comment:
That's the only thing that needs changing. Can you push a new version with this fix so we can get this merged?